Sources/Model/SmartHomeDevice.swift

```diff
@@ -54,6 +54,7 @@ extension FritzBox {
         public var identifier: String = ""
         public var features: Feature = []
         public var manufacturer: String = ""
+        public var present: Bool = false
```

Comment:
yes, no problem 🙂(but probably takes some days until i can do it)

I didn’t name it `isPresent`, because booleans in FritzBoxKit are
 ```
       public var batteryLow: Bool = false
       public var locked: Bool?
       public var lockedByDevice: Bool?
       public var state: Bool?
       public var lockedBySoftware: Bool?
```
        
but, as i said, no problem with name isPresent also

Reply:
Oh. You are right. Good point. So as this is a general issue I would merge the PR and think about a general renaming later.